content/docs/guide-development.md

```diff
@@ -117,7 +117,7 @@ https://renkulab.io/
 
 TODO: explain how to run server
 
-### Developing service in Renku
+### Developing plugins in Renku
```

Comment:
I think it was like just like that a couple of commits ago :)

Reply:
indeed, it was a mistake from some copy and paste, the paragraph is obviously about renku plugins

Reply:
Yes, and my comment was that we do not to teach developing renku plugins to domain expert workflow developer. We need to teach them how to use the plugins while developing services.